core/templates/components/common-layout-directives/common-elements/promo-bar.directive.ts

```diff
@@ -62,10 +62,11 @@ angular.module('oppia').directive('promoBar', [
             setPromoDismissed(true);
           };
           ctrl.$onInit = function() {
-            PromoBarService.getPromoBarData().then(function(promoBarObject) {
-              ctrl.promoBarIsEnabled = promoBarObject.promoBarEnabled;
-              ctrl.promoBarMessage = promoBarObject.promoBarMessage;
-            });
+            PromoBarBackendApiService.getPromoBarDataAsync()
+              .then(function(promoBarObject) {
+                ctrl.promoBarIsEnabled = promoBarObject.promoBarEnabled;
+                ctrl.promoBarMessage = promoBarObject.promoBarMessage;
+              });
```

Comment:
Optional: promoBarObject --> promoBar. (Though this is not an expected change for this PR, I think it's a minor fix and would make the code better.)

Reply:
Done!